grails-app/domain/org/pih/warehouse/order/OrderAdjustment.groovy

```diff
@@ -59,9 +58,13 @@ class OrderAdjustment implements Serializable {
 
     static belongsTo = [order: Order, orderItem: OrderItem]
 
+    static hasMany = [invoiceItems: InvoiceItem]
```

Comment:
I don't love adding new relationships that don't make sense. I don't understand why we want to do this. We're creating a new join table? How are we populating that join table? 

I'm clearing missing something but here so I suggest we move this to a tech huddle.

Reply:
It is not new. It was already on the `InvoiceItem` before as:
`static hasMany = [shipmentItems: ShipmentItem, orderItems: OrderItem, orderAdjustments: OrderAdjustment]`
and there are already join tables for these, these were just not used on the order items, order adjustments, and shipment items.